airflow/providers/airbyte/sensors/airbyte.py

```diff
@@ -54,7 +54,7 @@ def __init__(
 
     def poke(self, context: dict) -> bool:
         hook = AirbyteHook(airbyte_conn_id=self.airbyte_conn_id, api_version=self.api_version)
-        job = hook.get_job(job_id=self.airbyte_job_id)
+        job = hook.get_job(job_id=int(self.airbyte_job_id))
```

Comment:
We shouldn't need to cast the job id to an int here -- if we do it's a breaking change.

```suggestion
        job = hook.get_job(job_id=self.airbyte_job_id)
```

Reply:
`def get_job(self, job_id: int) -> Any:`
This is fixing this error, sorry not sure why its a breaking change.

airflow/providers/airbyte/sensors/airbyte.py:57: error: Argument "job_id" to "get_job" of "AirbyteHook" has incompatible type "str"; expected "int"

Reply:
Looks like `airbyte_job_id` in the sensor's constructor is still typed as a `str` rather than `int`. Changing the typing there should remove the need for the cast.

Reply:
done.